srcpkgs/evolution/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'evolution'
 pkgname=evolution
 version=3.38.0
-revision=1
+revision=2
```

Comment:
I don't think we need to rev-bump. The existence of gtkhtml-devel doesn't affect result binary.

Reply:
Nevermind, I was wrong. We need to bump to update dependency for `-devel`.